06_file_organization_01_organizing_projects.Rmd

```diff
@@ -0,0 +1,156 @@
+
+Naming and organizing files seems very boring, but it one of the most important parts of any data science project! Not having the files or the data available is one of the most common reasons that errors are missed in data science projects.
+
+### A project organization framework
+
+We will set up data science projects on RStudio Cloud. Open your web browser and navigate to the website [https://rstudio.cloud/](https://rstudio.cloud/).
+
+
+![Go to RStudio Cloud](https://docs.google.com/presentation/d/1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70/export/png?id=1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70&pageid=g313ecd4576_0_79)
+
+
+Then log in and click on the project *my_first_project*.
+
+
+![Click on my_first_project](https://docs.google.com/presentation/d/1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70/export/png?id=1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70&pageid=g313ecd4576_0_2)
+
+
+
+You should now see a screen that looks like this where there are three windows. The window in the lower right hand corner of the screen is the part that shows all the files you have in your project. Right now there shouldn't be any files since we just created this project in the last lesson but didn't do anything in the project.
+
+
+![The RStudio Cloud interface showing the empty my_first_project](https://docs.google.com/presentation/d/1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70/export/png?id=1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70&pageid=g313ecd4576_0_155)
+
+
+
+Each time you start a new project you will need to create a set of folders for that project. You can create a new folder by clicking on New Folder at the top of the file window.
+
+
+![Click New Folder to create a new folder.](https://docs.google.com/presentation/d/1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70/export/png?id=1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70&pageid=g313ecd4576_0_161)
+
+
+Then you can type in the name of the folder you want to create. First let's create a folder called _data_.
+
+
+![Name the folder data](https://docs.google.com/presentation/d/1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70/export/png?id=1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70&pageid=g313ecd4576_0_168)
+
+
+You should now see a folder called data in the file window.
+
+
+
+![We now have a data folder in the files in the project.](https://docs.google.com/presentation/d/1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70/export/png?id=1jNeIkKjyVenNF5AEqNpspnLuKXSiXbg-I6VC_uy-b70&pageid=g313ecd4576_0_173)
+
+Next we will create a few more folders. For each one click the New Folder button, enter the name and click ok.
+
+Different projects may call for different organization schemes and folder set ups. The rules we give you here may not fit **every** project you work on. But this organizational scheme is a good start that you can iteratively evaluate and change as needed as you continue to work on your project. 
```

Comment:
Added a little more nuance here and tried to make the folder recommendations a little more widely applicable. 

I also didn't agree with the whole "raw" vs "final" versions of code because that is what version control is for.